template/FullStack/React(Frontend)+Nodejs(Backend)/docker-compose.yaml

```
@@ -0,0 +1,38 @@
+      dockerfile: Dockerfile
+    container_name: univ_template_nodejs_server
+    depends_on:
+      - mongodb
+    ports:
+      - "3000:3000"
+    environment:
+       MONGO_URL: mongodb://mongodb_uni:27017/uni
+       JWT_SECRET: jwt_secret
+
+  frontend:
+    build: 
```

yamllint

[error] trailing spaces

(trailing-spaces)